### PR TITLE
feat(config-ui): add bp upgrade error

### PR DESCRIPTION
**Pull request by mintsweet**
### Summary
1. fix missed fields on Jira.
2. add bp **upgrade** error.

### CODE REVIEW

File: config-ui/src/pages/blueprint/detail/use-detail.ts [hunk 1]
```diff
@@ -55,6 +57,14 @@ export const useDetail = ({ id }: UseDetailProps) => {
     setLoading(true)
     try {
       const res = await API.getBlueprint(id)
+
+      // need to upgrade 2.0.0
+      if (res.settings.version === '1.0.0') {
```
Comment (likyh) [inline config-ui/src/pages/blueprint/detail/use-detail.ts:62]: It looks like an error may trigger when opening bp detail page. But how to run the existing v100 bp?

File: config-ui/src/error/components/bp-upgrade/use-bp-upgrade.ts [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+      const res = await API.getJIRA(prefix, options.boardId)
+      return {
+        connectionId,
+        boardId: res.id,
+        name: res.name,
+        self: res.self,
+        type: res.type,
+        projectId: res?.location?.projectId
+      }
+    }
```
Comment (likyh) [inline config-ui/src/error/components/bp-upgrade/use-bp-upgrade.ts:93]: Jenkins may not have the scope, and we need to add some tips to lead users to select scopes.

File: config-ui/src/pages/blueprint/detail/use-detail.ts (hunk 1 above)
Comment (mintsweet) [reply config-ui/src/pages/blueprint/detail/use-detail.ts:62]: The user cannot continue to operate the bp v100 on the config ui.

Comment (likyh) [reply config-ui/src/pages/blueprint/detail/use-detail.ts:62]: but user need to run it

**Review by likyh — APPROVED** [review]
Just merged for test

**Pull request merged**